CallsheetLocalizations/uk.xcloc/Localized Contents/uk.xliff

```diff
@@ -37,1064 +37,1065 @@
         <note/>
       </trans-unit>
       <trans-unit id="#%lld" xml:space="preserve">
-        <source>#%lld</source>
+        <source>#%lld</source><target>#%lld</target>
         <note/>
       </trans-unit>
       <trans-unit id="#%llu" xml:space="preserve">
-        <source>#%llu</source>
+        <source>#%llu</source><target>#%llu</target>
         <note/>
       </trans-unit>
       <trans-unit id="%@" xml:space="preserve">
-        <source>%@</source>
+        <source>%@</source><target>%@</target>
         <note/>
       </trans-unit>
       <trans-unit id="%@ (%@)" xml:space="preserve">
-        <source>%1$@ (%2$@)</source>
+        <source>%1$@ (%2$@)</source><target>%1$@ (%2$@)</target>
         <note/>
       </trans-unit>
       <trans-unit id="%@ - %@" xml:space="preserve">
-        <source>%1$@ - %2$@</source>
+        <source>%1$@ - %2$@</source><target>%1$@ - %2$@</target>
         <note/>
       </trans-unit>
       <trans-unit id="%@ / %@" xml:space="preserve">
-        <source>%1$@ / %2$@</source>
+        <source>%1$@ / %2$@</source><target>%1$@ / %2$@</target>
         <note/>
       </trans-unit>
       <trans-unit id="%@ by %@" xml:space="preserve">
-        <source>%1$@ by %2$@</source>
+        <source>%1$@ by %2$@</source><target>%1$@ від %2$@</target>
```

Comment:
@cliss where is this string used?

Reply:
<img width="288" alt="Screenshot 2024-05-13 at 9 47 17 AM" src="https://github.com/cliss/callsheet-localizations/assets/282460/9334c1d1-6c0e-4d30-a6f8-33648cdadbd6">

`{Icon name} by {Icon artist}`